src/EdFi.DataManagementService.sln

```diff
@@ -5,8 +5,6 @@ VisualStudioVersion = 17.4.33122.133
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "services", "services", "{22F7EF40-C77C-4AF0-8C1E-500EA6B85496}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EdFi.DataManagementService.Api.Tests.E2E", "services\EdFi.DataManagementService.Api.Tests.E2E\EdFi.DataManagementService.Api.Tests.E2E.csproj", "{678F06E1-CC23-438A-9CDF-52CA4C24BCED}"
-EndProject
```

Comment:
I'm not sure about this. Please ask one or two others on the team for feedback on this.

Reply:
Sure, I moved the entire folder and for some reason, the references to the E2E project were not updated in the .sln file.

Reply:
@stephenfuqua I will close this PR until DMS-196 is completed. We discussed possible solutions for moving the folders, such as creating a new project and moving the required folders/files instead of the whole project, or trying with Visual Studio instead of Visual Studio Code.

Reply:
All you need to do is re-add the project to the solution, from within Visual Studio or Visual Studio Code.